src/Cuboid.h

```diff
@@ -20,15 +20,14 @@ class cCuboid
 		cCuboid(const cCuboid & a_Cuboid);
 	#endif
 
-	// DEPRECATED, use cCuboid(Vector3i, Vector3i) instead
-	cCuboid(int a_X1, int a_Y1, int a_Z1, int a_X2, int a_Y2, int a_Z2) : p1(a_X1, a_Y1, a_Z1), p2(a_X2, a_Y2, a_Z2)
-	{
-		LOGWARNING("cCuboid(int, int, int, int, int, int) constructor is deprecated, use cCuboid(Vector3i, Vector3i) constructor instead.");
-	}
+	// tolua_end
+	// Exported in ManualBindings.cpp to support the old deprecated coord-based overload.
 
-	void Assign(int a_X1, int a_Y1, int a_Z1, int a_X2, int a_Y2, int a_Z2);
+	void Assign(const Vector3i & a_Point1, const Vector3i & a_Point2);
```

Comment:
```suggestion
	void Assign(Vector3i a_Point1, Vector3i a_Point2);
```

Reply:
Why is it better to make a copy?

Reply:
See https://github.com/cuberite/cuberite/issues/3853#issuecomment-318932036

Reply:
Alright, I'll change that. I assume that works only for small `Vector3i`s, not for the larger `Vector3f`, `Vector3d` or `cCuboid` instances, right?

Reply:
See [CONTRIBUTING.md] for the guidelines we agreed upon at the time.
> - Parameters smaller than 4 elements (e.g. Vector3, cChunkCoords) should be passed by value. All other parameters should be passed by const reference, where applicable.
>   - `Foo(Vector3d a_Param1, const cCuboid & a_Param2)`
>   - See the discussion in issue #3853


[CONTRIBUTING.md]: https://github.com/cuberite/cuberite/blob/master/CONTRIBUTING.md#L28-L30

Reply:
Thanks for the pointers. I got a bit rusty by not working on Cuberite for a long time :)